hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/RDBStore.java

```diff
@@ -94,6 +97,14 @@ public RDBStore(File dbFile, DBOptions options,
     this.writeOptions = writeOptions;
 
     try {
+      List<TableConfig> columnFamiliesInDb = getColumnFamiliesInExistingDb();
+      List<TableConfig> extraCf = columnFamiliesInDb.stream().filter(
+          cf -> !families.contains(cf)).collect(Collectors.toList());
+      for (TableConfig family : extraCf) {
+        LOG.info("Adding column family {}", family.getName());
```

Comment:
Can this comment be more clear saying ColumnFamily in DB, but not in the current version something like that?

Reply:
Will do.